sup3r/utilities/stitching.py

```diff
@@ -0,0 +1,420 @@
+"""Utilities for stitching south east asia domains"""
+# -*- coding: utf-8 -*-
+# pylint: disable=all
```

Comment:
Is there a good reason not to lint this file? It actually looks like you did a great job with formatting and whatnot.

Reply:
I did this so I didn't have to add xesmf to setup but your suggestion to move to init is a good idea